src/screens/NotePropertiesScreen.tsx

```diff
@@ -52,7 +52,7 @@ export default function NotePropertiesScreen(props: PropsType) {
       .sort((a, b) => (a.meta!.name!.toUpperCase() >= b.meta!.name!.toUpperCase()) ? 1 : -1)
       .values()
   );
-  const [collection, setCollection] = React.useState<typeof options[0] | undefined>();
+  const [collection, setCollection] = React.useState<typeof options[0] | undefined>(options.length > 0 ? options[0] : undefined);
```

Comment:
Can probably just be:
`   const [collection, setCollection] = React.useState(options[0]);`
The type is inferred from the default value, and options[0] is either the value of undefined.

Reply:
I agree that the type casting is not necessary anymore, but won't `options[0]` throw an error if `options` is empty?

Reply:
Nope.
```
% node
Welcome to Node.js v15.5.0.
Type ".help" for more information.
> [][0]
undefined
```

Reply:
Oh great!

Reply:
Actually now type is wrong because it infers that it's always defined.

Reply:
OK, so keep the options.length but just remove the type inference.